### DISCUSSION TITLE
setup with cloudflare tunnel hint

### DISCUSSION THREAD

**Discussion by Kugeleis** [nextcloud/all-in-one#4382] [Questions] [answered]
I set up the AIO on a home server accessed via a cloudflare tunnel. After hours of trial and error despite following the reverse proxy setup guide I found something nowhere mentioned:

I had to point the cloudflare tunnel URL to the aio-docker-network apache IP address like so: `http://172.26.0.10:11000`.

**Comment by szaimen** [comment, marked as answer]
Hi, can you follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md#6-how-to-debug-things?
